### PR TITLE
Updates to setup tool README

### PR DESCRIPTION
**Pull request by rajivnathan**

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Review by ranakan19 — APPROVED** [review]

File: setup/README.adoc [hunk 1]
```diff
@@ -154,10 +154,10 @@ make clean-e2e-resources
 
 1. Install operators
 ```
-go run setup/main.go --users 1 --default 1 --custom 0 --username setup
+go run setup/main.go --users 1 --default 1 --custom 0 --username baseline --testname=baseline
 ```
 
 2. Run setup for 2000 users
 ```
-go run setup/main.go --users 2000 --default 2000 --custom 0 --username pizza
```
Comment (ranakan19) [inline setup/README.adoc:162]: no more pizza :(

File: setup/README.adoc [hunk 2]
```diff
@@ -36,7 +36,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OpenShiftSDN
+  networkType: OVNKubernetes
```
Comment (mfrancisc) [inline setup/README.adoc:39]: just curious - why this change ? I'm still using `OpenShiftSDN` for my "local" development. Did something changed in ocp 4.15 ?

**Review by mfrancisc — APPROVED** [review]

File: setup/README.adoc (hunk 2 above)
Comment (mfrancisc) [reply setup/README.adoc:39]: Ha got it! Thanks for the heads up. I still need to switch to 4.15 locally so that's why I haven't faced this issue.

Comment (mfrancisc) [reply setup/README.adoc:39]: Also last times I've used ClusterBot to generate Rosa cluster, which turned out to be quicker to provision and more "stable" in kind of installation process.

Comment (MatousJobanek) [reply setup/README.adoc:39]: :thinking: but our clusters are still running OpenShiftSDN, right? Or did they started migrating the already existing clusters to OVNKubernetes? IIRC, the migration was no-go for the already existing OSD clusters some time ago, so I'm curious if this has changed as well or not.

Comment (rajivnathan) [reply setup/README.adoc:39]: Yeah, my understanding is that migration to OVNKubernetes is still not supported for OSD clusters. At least according to the 4.15 docs: https://docs.openshift.com/container-platform/4.15/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html

We should check with the OSD team though. I mentioned it to @alexeykazakov already.
